src/styles/breakpoints.js

```diff
@@ -43,5 +51,22 @@ export default function createBreakpoints(
     return breakpoints[name];
   }
 
-  return { keys, values, up, down, only, getWidth };
+  let matched = priority
+    .reduce((a, k) => {
+      const media = window.matchMedia(breakpoints[k]);
```

Comment:
My biggest concern is regarding having a correct layout during the first paint. And not waiting for the js to load in order to get the correct one. As far as I know, we can't use the `matchMedia` API with this constrainte.
What are we using it for? Why not using the CSS media queries?